data/test/vtgate/select_cases.txt

```diff
@@ -543,3 +543,30 @@
     "Values": 5
   }
 }
+
```

Comment:
i'm guessing that this comment is actually outdated by our conversation on wednesday, no? as things are now in the parser, a query like:
`select * from select col1, col2 from user union all select col1, col2 from user_extra`
is greeted by:
`syntax error at position 21 near 'select'`
and
`select id, name from user where id in (select id from user union all select user_id from user_extra)`
bombs out with
`unsupported: complex join in subqueries`